src/Gregwar/Captcha/CaptchaBuilder.php

```diff
@@ -257,8 +269,18 @@ public function setBackgroundImages(array $backgroundImages)
      */
     protected function drawLine($image, $width, $height, $tcol = null)
     {
+        $red = $this->rand(100, 255);
```

Comment:
`rand` is called unnecessarily when the line is set here, this will cause the fingerprint to contain useless data